doc/plugins.md

```diff
@@ -91,6 +91,9 @@ The list of plugins:
     — turn some or all remark’s tokenizers on or off
 *   🟢 [`remark-directive`](https://github.com/remarkjs/remark-directive)
     — new syntax for directives (generic extensions)
+*   🟢 [`remark-directive-rehype`](https://github.com/IGassmann/remark-directive-rehype)
+    — enable Markdown directives to be parsed as HTML with [`remark-directive`][d] and
+    [`remark-rehype`](https://github.com/remarkjs/remark-rehype)
```

Comment:
How about something like: `turn [directives][d] into HTML custom elements (rehype compatible)` instead?
A bit shorter, and matches the rest of the plugins when referencing rehype?